File: app/src/main/java/com/nollpointer/dates/api/Mobile.java
package com.nollpointer.dates.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Mobile {

    @SerializedName("page")
    @Expose
    private String page;

    public String getPage() {
        return page;
    }

    public void setPage(String page) {
        this.page = page;
    }

}

File: app/src/main/java/com/nollpointer/dates/model/Mobile.kt
package com.nollpointer.dates.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author Onanov Aleksey (@onanov)
 */
class Mobile {
    @SerializedName("page")
    @Expose
    var page: String? = null

}